index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/styles/css/styles.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <div class="nav">
+        <div class="nav__row">
+          <div class="nav__column">
+            <div class="nav__logo logo">
+              <div class="logo__image">
+                <img src="./assets/images/png/mail (image).png" alt="Logo" />
+              </div>
+              <div class="logo__text">NinjaMail</div>
+            </div>
+          </div>
+
+          <div class="nav__column">
+            <div class="nav__menu">
+              <div class="nav__item">
+                <a class="nav__item__item_text" href="">Features</a>
+              </div>
+              <div class="nav__item">
+                <a class="nav__item__item_text" href="">Pricing</a>
+              </div>
+              <div class="nav__item">
+                <a class="nav__item__item_text" href="">Services</a>
+              </div>
+              <div class="nav__item">
+                <a class="nav__item__item_text" href="">Partners</a>
+              </div>
+              <div class="nav__item">
+                <div class="green_button">
+                  <a class="nav__item__item_text">Sign up free</a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="content">
```

Comment:
To divide some section for parent tag, it's better to use tag SECTION for semantic